Source/GSThemeDrawing.m

```diff
@@ -2073,6 +2072,38 @@ - (void) drawWindowBorder: (NSRect)rect
     }
 }
 
+- (void) setFrameForCloseButton: (NSButton *)closeButton
```

Comment:
Not your fault, but we shouldn't need this second method to determine the close button size.